python-test-samples/apigw-lambda/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+    "title": "Python Test Samples Starter Project",
```

Comment:
Can you call this Python Starter Project

Reply:
Changed in latest commit